### ISSUE TITLE
Docs Index: full text is no longer picked up

### ISSUE THREAD

**Issue by trieloff** [adobe/helix-website#334]
https://docs.google.com/spreadsheets/d/1CO0a9cbugesnfdgQXWH6ZuP5PHLatENayX37EfT7hrA/edit#gid=0

The `content` column now only contains "style".

**Comment by dominique-pfister** [comment]
Not quite: it _starts_ with "style", but then comes a LF and the complete contents following:

<img width="1938" alt="Screenshot 2023-07-03 at 09 11 10" src="https://github.com/adobe/helix-website/assets/22323322/527cfd3c-00a9-4f5d-813c-37e9b2a9d3c7">

**Comment by trieloff** [comment]
Ah, so then it's ok.
